### PR TITLE
Removed getProps/getElementProps entirely

### PR DESCRIPTION
**Pull request by nitzanyiz**
## Description
Removed the usage of getProps from the basic component driver. To get the element props we now need to use getElement().props.
Changed Text driver getText to return text if it has only one children.

## Changelog
None.

## Additional info
MADS-4058

### CODE REVIEW

File: src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx [hunk 1]
```diff
@@ -4,8 +4,8 @@ import {WheelPickerDriver} from '../WheelPicker/WheelPicker.driver';
 import {SectionsWheelPickerProps} from './index';
 
 export const SectionsWheelPickerDriver = (props: ComponentProps) => {
-  const driver = useComponentDriver<SectionsWheelPickerProps>(props);
-  const sections = driver.getProps().children as SectionsWheelPickerProps;
+  const driver = useComponentDriver(props);
+  const sections = driver.getElement().children as SectionsWheelPickerProps;
```
Comment (M-i-k-e-l) [inline src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx:8]: Why do we need the `SectionsWheelPickerProps`?

**Review by M-i-k-e-l — COMMENTED** [review]
One small thing

File: src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx (hunk 1 above)
Comment (nitzanyiz) [reply src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx:8]: This is needed so TS will enable us to get the testID from the children. This is not related to this PR though.

Comment (M-i-k-e-l) [reply src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx:8]: I don't see a usage of `driver.sections` in private, and I don't see an error in public...

Comment (nitzanyiz) [reply src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx:8]: Its used here
https://github.com/wix/react-native-ui-lib/blob/43151d5a2e129c3fc9f39bbb8c1fefbb63366536/src/components/sectionsWheelPicker/__tests__/index.spec.tsx#L33-L39

Comment (M-i-k-e-l) [reply src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx:8]: I still don’t understand why we need the type

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**